packages/api/src/controllers/asset.ts

```diff
@@ -247,32 +247,39 @@ const fieldsMap: FieldsMap = {
   playbackId: `asset.data->>'playbackId'`,
   "user.email": { val: `users.data->>'email'`, type: "full-text" },
   meta: `asset.data->>'meta'`,
+  ipfsVideoFileCid: `asset.data->'status'->'storage'->'ipfs'->'data'->'videoFileCid'`,
+  ipfsNftMetadataCid: `asset.data->'status'->'storage'->'ipfs'->'data'->'nftMetadataCid'`,
 };
 
 app.get("/", authorizer({}), async (req, res) => {
-  let { limit, cursor, all, allUsers, order, filters, count } = toStringValues(
-    req.query
-  );
+  let { limit, cursor, all, allUsers, order, filters, playbackId, count } =
+    toStringValues(req.query);
   if (isNaN(parseInt(limit))) {
     limit = undefined;
   }
   if (!order) {
     order = "updatedAt-true,createdAt-true";
   }
 
-  if (req.user.admin && allUsers && allUsers !== "false") {
-    const query = parseFilters(fieldsMap, filters);
-    if (!all || all === "false") {
-      query.push(sql`asset.data->>'deleted' IS NULL`);
-    }
+  const query = parseFilters(fieldsMap, filters);
+  if (playbackId) {
+    query.push(sql`asset.data->>'playbackId' = ${playbackId}`);
+  }
+  if (!all || all === "false") {
+    // TODO: Consider keeping this flag as admin-only
```

Comment:
I agree this would be better as admin only, at least for consistency with the other apis that never allows the user to retrieve the `deleted` objects